devops/docker/idea-image/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM cimg/openjdk:11.0
+
+RUN sudo apt update && sudo apt install -y wget
+
+# ENV is transferred to container
+ENV IDEA_VERSION="ideaIC-2022.2.2"
+ENV IDEA_PATH="/home/circleci/idea"
+ENV PATH="${PATH}:${IDEA_PATH}/bin"
+
+# ARG is not transferred to container
+ARG IDEA_DOWNLOAD_LINK="https://download.jetbrains.com/idea/${IDEA_VERSION}.tar.gz"
+
+RUN mkdir -p ${IDEA_PATH}
+WORKDIR $IDEA_PATH
+
+RUN wget -qO- $IDEA_DOWNLOAD_LINK | \
+    tar xvz -C $IDEA_PATH --strip-components=1 && \
```

Comment:
>strip-components

I am not sure why we need this ?
We have variable for full path, depth of path does not matter.

Reply:
Stripping outer directory allows us to have an easier path to deal with and less diff in the future

Reply:
We just need to adjust `ENV IDEA_PATH="/home/circleci/idea"`
To have longer path.

Reply:
In the future, we can simply update IDEA_VERSION in this file and build/tag the image.

Reply:
Can you share path without strip ? To let me see a problem

Reply:
```
➜  contribution git:(add-idea-docker-image) ✗ ls test 
idea-IC-222.4167.29
```
it is not simple version number

Reply:
If we do not strip, future update will require manual download, check directory name, then change it in dockerfile

Reply:
Oh, yes this name we do not need